packages/l/libreswan/package.yml

```diff
@@ -22,11 +22,16 @@ builddeps  :
 conflicts  :
     - strongswan
 environment: |
-    export INC_USRLOCAL=/usr
+    export PREFIX=/usr
+    export USE_DNSSEC=false
     export WERROR_CFLAGS=
-setup      : |
-    %apply_patches
 build      : |
     %make programs 
 install    : |
-    %make_install FINALMANDIR=/usr/share/man
+    %make_install MANDIR=/usr/share/man \
+    SYSCONFDIR=/usr/share/defaults/etc \
```

Comment:
Like I said in #7140, I suspect this will require a patch to work properly.

Reply:
Same as in Strongswan, I need this more than we need proper stateless. If you could remove the last `/etc/` in the `/usr/share/defaults/etc`, I'll call it good.